backend/apps/slack/models/google_auth.py

```diff
@@ -0,0 +1,125 @@
+"""Slack Google OAuth Authentication Model."""
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils import timezone
+
+from apps.common.clients import get_google_auth_client
+from apps.slack.models.member import Member
+
+AUTH_ERROR_MESSAGE = (
+    "Google OAuth client ID, secret, and redirect URI must be set in environment variables."
+)
+
+
+class GoogleAuth(models.Model):
+    """Model to store Google OAuth tokens for Slack integration."""
+
+    member = models.OneToOneField(
```

Comment:
Okay, so this is going to be used from Slack application instead of Nest UI? Trying to figure out the flow based on the initial models.